scope_registry.go

```diff
@@ -34,9 +34,9 @@ var (
 
 	// Metrics related.
 	internalTags             = map[string]string{"version": Version}
-	counterCardinalityName   = "tally.internal.counter-cardinality"
-	gaugeCardinalityName     = "tally.internal.gauge-cardinality"
-	histogramCardinalityName = "tally.internal.histogram-cardinality"
+	counterCardinalityName   = "tally-internal-counter-cardinality"
```

Comment:
I don't think either dots or dashes are accepted by prometheus reporter. At least they not legal characters according to [Prom data model](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels)

Reply:
will change to underscore